state/upgrades_test.go

```diff
@@ -1005,58 +1005,11 @@ func (s *upgradesSuite) makeModel(c *gc.C, name string, attr testing.Attrs) *Sta
 }
 
 func (s *upgradesSuite) TestAddStatusHistoryPruneSettings(c *gc.C) {
-	settingsColl, settingsCloser := s.state.getRawCollection(settingsC)
-	defer settingsCloser()
-	_, err := settingsColl.RemoveAll(nil)
-	c.Assert(err, jc.ErrorIsNil)
-
-	m1 := s.makeModel(c, "m1", testing.Attrs{
-		"max-status-history-age":  "96h",
-		"max-status-history-size": "4G",
-	})
-	defer m1.Close()
-
-	m2 := s.makeModel(c, "m2", testing.Attrs{})
-	defer m2.Close()
-
-	err = settingsColl.Insert(bson.M{
-		"_id": "someothersettingshouldnotbetouched",
-		// non-model setting: should not be touched
-		"settings": bson.M{"key": "value"},
-	})
-	c.Assert(err, jc.ErrorIsNil)
-
-	cfg1, err := m1.ModelConfig()
-	c.Assert(err, jc.ErrorIsNil)
-	expected1 := cfg1.AllAttrs()
-	expected1["resource-tags"] = ""
-
-	cfg2, err := m2.ModelConfig()
-	c.Assert(err, jc.ErrorIsNil)
-	expected2 := cfg2.AllAttrs()
-	expected2["max-status-history-age"] = "336h"
-	expected2["max-status-history-size"] = "5G"
-	expected2["resource-tags"] = ""
-
-	expectedSettings := bsonMById{
-		{
-			"_id":        m1.ModelUUID() + ":e",
-			"settings":   bson.M(expected1),
-			"model-uuid": m1.ModelUUID(),
-		}, {
-			"_id":        m2.ModelUUID() + ":e",
-			"settings":   bson.M(expected2),
-			"model-uuid": m2.ModelUUID(),
-		}, {
-			"_id":      "someothersettingshouldnotbetouched",
-			"settings": bson.M{"key": "value"},
-		},
-	}
-	sort.Sort(expectedSettings)
+	s.checkAddPruneSettings(c, "max-action-age", "max-action-size", AddStatusHistoryPruneSettings)
+}
 
-	s.assertUpgradedData(c, AddStatusHistoryPruneSettings,
-		expectUpgradedData{settingsColl, expectedSettings},
-	)
+func (s *upgradesSuite) TestAddActionPruneSettings(c *gc.C) {
+	s.checkAddPruneSettings(c, "max-action-age", "max-action-size", AddActionPruneSettings)
```

Comment:
Sorry to raise this now - I think this should be "max-action-results-age" and "max-action-results-size"
Could you please rename as a driveby in this PR